src/components/component-links.tsx

```diff
@@ -81,9 +94,8 @@ function ComponentLinks(props: ComponentLinksProps) {
   const npmLink = npm?.package && (
     <ComponentLink
       url={`https://www.npmjs.com/package/${npm.package}`}
-      icon={FaNpm}
-      iconSize="2rem"
-      iconColor="red.500"
+      icon={<FaNpm />}
+      name="NPM"

```

Comment:
We could use an Enum for those keys. It would make the solution more robust.

Reply:
I agree, will do!